src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -121,6 +121,7 @@
 #undef USE_HAL_DRIVER
 
 #include "device/dcd.h"
+#include "bsp/board.h"
```

Comment:
How do you suggest implementing a delay in the driver?  In order to send a remote wakeup on this hardware, one has to assert a bit, wait 1 to 15 ms, then deassert it.

Reply:
please remove it, the `tinyusb/src` is self-contained, it won't need anything else except those register definitions. If we have no solution, we either drop the feature or re-organize the stack or API. But I think we can address this.

Reply:
There are not SOF during suspend mode, so we can't use USB timing. Perhaps a busy-loop looking at the systick counter values... assuming it is set for 1 kHz. I'll continue to think about it.

Reply:
@pigrew systick is only available for ARM, and we are requiring application to enable that. There is tickless system like one I am working with on BLE repo, that is not really cross-platform approach. Check out my comment below, it may work out.